src/gordon_gcp/plugins/service/gdns_publisher.py

```diff
@@ -298,6 +299,39 @@ def _format_resource_record_changes(self, action, resource_record):
 
         return False
 
+    async def _dispatch_changes(
+            self, resource_record, action, logger):
```

Comment:
I know its defined in the spec, but I'd prefer is we could do the line breaking at least to a de-facto pattern. I think we tend to start listing the args on the same line as the method declaration.

Reply:
... mumble [yapf](https://github.com/google/yapf) mumble mumble ... :)

Reply:
Ha, it actually fits on one line now.